cadc-test-vos/src/main/java/org/opencadc/conformance/vos/NodesTest.java

```diff
@@ -965,28 +965,47 @@ public void testPermissions() throws Exception {
         String childPath = parentName + "/" + childName;
         final VOSURI childURI = getVOSURI(childPath);
         final URL childURL = getNodeURL(nodesServiceURL, childPath);
+
```

Comment:
test setup:

/int-tests/testPermissions
/int-tests/testPermissions/testGroupUser
/int-tests/testPermissions/testListUser

The point of naming it `testPermissions` is so only this method does the setup. it looks like you made a new child node (testListUser) instead of using the one that was there... the names make it sound like it is intentional but I can't grok what for. The cleanup only removes 2 of the 3 nodes.

What is "testListUser" for and why is it named that way?

Why do you need more than /int-tests/testPermissions/x where `x` can be any kind of node?

Reply:
The test creates the parent container node as the test user.
Gets the parent node as a second test user. Used to fail, now it succeeds.
Writes the child node to the parent as a second test user, which fails.
Gives the second test user write permissions to the parent node.
Writes the child to the parent as the second test user, which now succeeds.
Updates the child quota property as the test user.
Gets the child as the test user, which succeeds. Not sure what that is testing.

So the existing child node is owned by the second test user. 

Getting the parent as a user without read access to the parent should return the parent, because it's in the public int-test node, but not the child.
Given the existing child is put by the second test user, I would need a third test user to test with the existing child. Instead I added a second child written by the test user. The testListUser name is because the second test user should not be able to list the children of the parent.

I guess I could change the ownership of the existing child instead, and clear the read/write groups if needed. That seems overly complicated, and makes this test even harder to understand. Which maybe is a clue I should've written a separate test instead of adding on to this one?

I don't like any of the names in this test. The parent is referred to as parentName, testNode, or nodeURL, which I find confusing. And I don't know why a node has User in the name. But it makes sense to whomever wrote it, so I just go with the style, instead of rewriting it so that it makes better sense to me.

Reply:
The only thing that changed is that some read requests are now allowed that used to be rejected.
So why test write permission is rejected? There are tests for that already. Anything extra just obscures the intent of the test. So again, 

**Why do you need more than /int-tests/testPermissions/x where x can be any kind of node?**